# input your gpt base url:
api_base: "xxx.com"
# input your gpt module:
gpt_module: "gpt-4-0125-preview"
# input your gpt api:
api_key: "xxxxxxxxxxxxxxx"
# input the timeout for gpt reply(default 20s):
time_out: 20

